backend/Application/Aggregates/Contract/Jobs/ContractDatabaseImportJob.cs

```diff
@@ -123,29 +123,18 @@ private async Task<long> GetFinalHeight(CancellationToken token)
         return finalHeight;
     }
     /// <summary>
-    /// Run each batch up to final height.
-    ///
-    /// Atomically get next batch interval from `_readCount`. If intervals get above <see cref="finalHeight"/> then
-    /// processing stops.
+    /// Run a batch.
     /// </summary>
-    private async Task RunBatch(long finalHeight, CancellationToken token)
+    private async ValueTask RunBatch(long height, CancellationToken token)
     {
-        while (!token.IsCancellationRequested)
-        {
-            using var _ = TraceContext.StartActivity(nameof(RunBatch));
+        using var _ = TraceContext.StartActivity(nameof(RunBatch));
             
-            var height = Interlocked.Increment(ref _readCount);
-            var blockHeightTo = height * _jobOptions.BatchSize;
-            if (blockHeightTo > finalHeight)
-            {
-                return;
-            }
-            var blockHeightFrom = Math.Max((height - 1) * _jobOptions.BatchSize + 1, 0);
-            var affectedRows = await DatabaseBatchImportJob((ulong)blockHeightFrom, (ulong)blockHeightTo, token);
+        var blockHeightTo = height * _jobOptions.BatchSize;
+        var blockHeightFrom = Math.Max((height - 1) * _jobOptions.BatchSize + 1, 0);
+        var affectedRows = await DatabaseBatchImportJob((ulong)blockHeightFrom, (ulong)blockHeightTo, token);
 
-            if (affectedRows == 0) continue;
-            _logger.Debug("Written heights {From} to {To}", blockHeightFrom, blockHeightTo);   
-        }
+        if (affectedRows == 0) return;
```

Comment:
I think you should change this to:
```
if (affectedRows == 0) { 
  return;
}
```
but I don't know what the general style for this repository has been. But later in this file a similar `if` statement is indeed done fully written out, and I believe this is better as all `if` statements will then be written lik that.